packages/trpc/server/routers/viewer/teams/hasTeamPlan.handler.ts

```diff
@@ -14,11 +14,6 @@ export const hasTeamPlanHandler = async ({ ctx }: HasTeamPlanOptions) => {
     where: {
       accepted: true,
       userId,
-      team: {
-        slug: {
-          not: null,
-        },
-      },
     },
```

Comment:
this would mean unpaid teams (unpublished = unpaid) would suddenly unlock paid only features